src/saddle_point.jl

```diff
@@ -618,12 +663,12 @@ accordingly.
    this struct will be reset to the restart point.
 - `current_primal_solution::AbstractVector`: If there is a restart then this
    vector might be set to the avg_primal_solution. However, it could remain
-   unchanged, if the algorithm thinks the averaged iterate is better than the
-   current iterate.
+   unchanged, if the algorithm thinks the current iterate is better than the
+   averaged iterate.
```

Comment:
Same comment as above. These changes should go on a separate PR.

Reply:
Moved to another PR